testing/scenario_app/bin/run_ios_tests.dart

```diff
@@ -49,7 +49,9 @@ void main(List<String> args) async {
   if (results.option('dump-xcresult-on-failure') case final String path) {
     dumpXcresultOnFailurePath = path;
   } else {
-    dumpXcresultOnFailurePath = io.Directory.systemTemp.createTempSync().path;
+    final dumpDir = io.Directory.systemTemp.createTempSync();
+    dumpXcresultOnFailurePath = dumpDir.path;
+    cleanup.add(() => dumpDir.delete(recursive: true));
```

Comment:
@matanlurey it needs to be deleted between the Skia and Impeller run though, not at the end of running both.

Reply:
Ah yes, you are right. Thank you!